readme.md

```diff
@@ -68,6 +68,7 @@ They also try to be very welcoming for 1st-time contributors.
 
 - [jest-extended](https://github.com/mattphillips/jest-extended)
 - [vue-dropzone](https://github.com/rowanwins/vue-dropzone)
+- [vue-dropzone](https://github.com/zeit/hyper)
```

Comment:
Why is the same label used? - Please add a description of the project and why is it awesome. There are hundreds projects out there. Why is project awesome? Did it accept more then 100 PRs? Does it provide good howtos?